### PR TITLE
Adding first attempt at CircleCI config.

### PR DESCRIPTION
**Pull request by zerebubuth**
I can't find a way to prototype this without a PR, so this is a PR to see if it kicks off a successful build.

### CODE REVIEW

**Comment by zerebubuth** [comment]
Connects to #492.

**Comment by zerebubuth** [comment]
@rmarianski could you review, please?

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+python-memcached==1.53
+PyYAML==3.11
+Shapely==1.4.3
+simplejson==3.6.4
+StreetNames==0.1.5
+Werkzeug==0.9.6
+wsgiref==0.1.2
+git+https://github.com/mapzen/TileStache@03c72f5#egg=TileStache
+git+https://github.com/mapzen/tilequeue@80b57d9#egg=tilequeue
+git+https://github.com/mapzen/tileserver@5babd91#egg=tileserver
```
Comment (rmarianski) [inline requirements.txt]: We won't want to point to master for tilestache/tilequeue/tileserver?

**Comment by zerebubuth** [comment]
Tests finally pass!

@rmarianski could you review, please?

File: requirements.txt (hunk 1 above)
Comment (zerebubuth) [inline requirements.txt]: Good point, fixed in 37c5cb5.

**Comment by rmarianski** [comment]
:+1:

**Pull request merged**